src/taipy/core/pipeline/pipeline.py

```diff
@@ -57,27 +56,27 @@ class Pipeline(_Entity, Submittable, _Labeled):
 
     def __init__(
         self,
-        config_id: str,
         properties: Dict[str, Any],
```

Comment:
Should we also remove the `properties`?

Since the PipelineConfig is gone, there will be no way to configure any properties. So it is kinda useless now.